tidb/manifest.json

```
@@ -29,7 +28,6 @@
       "spec": "assets/configuration/spec.yaml"
     },
     "dashboards": {
-      "overview": "assets/dashboards/one-screen-overview.json",
       "TiDB Overview": "assets/dashboards/overview.json"
```

```suggestion
      "overview": "assets/dashboards/overview.json"
```
This key cannot be changed, but don't worry, the name in the actual overview.json file will be used